test-cases/longevity/longevity-200GB-48h-network-monkey.yaml

```diff
@@ -14,6 +14,7 @@ nemesis_class_name: 'SisyphusMonkey'
 nemesis_include_filter: ['networking']
 nemesis_seed: '002'
 nemesis_interval: 10
+nemesis_multiply_factor: 15
```

Comment:
for this job, i'm not even sure 15 will be enough...
if the default "6" ran for ~ 10 hours (in a 48 hours job), we will need it to be at least some 30, but i would raise it to some 40 just to be sure

Reply:
15 is good enough.